server/constants/activities.ts

```diff
@@ -81,7 +81,7 @@ enum ActivityTypes {
   HOST_APPLICATION_CONTACT = 'host.application.contact',
   // Contributions
   CONTRIBUTION_REJECTED = 'contribution.rejected',
-  SUBSCRIPTION_ACTIVATED = 'subscription.activated',
+  SUBSCRIPTION_ACTIVATED = 'subscription.activated', // deprecated?
```

Comment:
Noticed that we're not using this anymore, is that by intention? Can move it down inside this enum into the activities that are marked as deprecated if so